portfolio/src/main/webapp/style.css

```diff
@@ -763,6 +763,11 @@ textarea {
   margin-right: 20px;
 }
 
+.uploadedImage {
```

Comment:
The CSS file is also quite large, if you wanted to break into into page section-related css files that you imported.

Reply:
it was starting to get super confusing, made separate css files for each section of website